arrow/arrow.py

```diff
@@ -387,7 +387,7 @@ def span(self, frame, count=1, bounds="[)"):
             whether to include or exclude the start and end values in the span. '(' excludes
             the start, '[' includes the start, ')' excludes the end, and ']' includes the end.
             If the bounds are not specified, the default bound '[)' is used.
-
+        :param exact: (optional) whether to return a timespan that starts exacly at specificed start
```

Comment:
The reason for the lint (docs really) command failing is because there needs to be a blank line between the params and the next part of the docstring.

When I add that in everything passes fine, try running `tox -e docs` or `make docs` on your machine to see if it passes.

Reply:
There are also some typos in this line, why don't you use something similar to the `span_range` docstring?

Reply:
Got it! I fixed these in the latest commit.